ui/transformation.py

```diff
@@ -118,10 +121,17 @@ def set_spinbox_ranges(self):
             spinbox.setRange(-10000000, 10000000)
             spinbox.setDecimals(5)
 
-    def retranslateUi(self, Transformation):
-        Transformation.setWindowTitle(
+    @staticmethod
+    def _make_text_bold(label: QLabel):
+        font = label.font()
+        font.setBold(True)
+        label.setFont(font)
+
+    @staticmethod
+    def retranslate_ui(transformation):
```

Comment:
Same as above, let's not change the name in this PR, and instead do a "global" replace across the entire code base in one PR.